pkg/cmd/pulumi/state/state_reprotect.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016-2025, Pulumi Corporation.
```

Comment:
```suggestion
// Copyright 2025, Pulumi Corporation.
```
Similarly if we've done this elsewhere